Makefile

```diff
@@ -150,7 +150,7 @@ constraints.txt:
 	pip-requirements.txt \
 	external/fv3kube/setup.py \
 	external/fv3fit/setup.py \
-	external/*.requirements.in \
+	external/fv3gfs-wrapper.requirements.in \
```

Comment:
Why remove the glob? I had intended for any `*.requirements.in` to work when setting this up...would be nice to leave that behavior in place if add any more submodules.

Reply:
Sure, good point